README.md

```diff
@@ -39,12 +35,19 @@ You can see the [full history of the Python API on the documentation site](http:
 ## Updating HTTPLib2
 
 1. Download the latest version of HTTPLib2 at https://pypi.org/project/httplib2.
-2. Extract the python2/httplib2 into shotgun_api3/lib/http2lib without the test folder.
+2. Extract the python2/httplib2 into shotgun_api3/lib/http2lib/python2 without the test folder.
+2. Extract the python3/httplib2 into shotgun_api3/lib/http2lib/python3 without the test folder.
 3. Scan the files for any references to importing httplib2 and make sure they import "from ." instead of "from httplib2" because the library isn't in the Python path.
 
-## Tests 
+## Maintaining Python 2 and 3 compatibility
+
+python-api should remain compatible with both Python 2, and 3.  To make this easier, we use [six](https://six.readthedocs.io/).  When adding code the works with types that have changed between Python 2 and 3, notably strings and files, it's advisable to use the `six` types for casting and comparisons. Be sure to follow Python 2 and 3 compatible conventions in code, especially when raising or capturing exceptions and printing. While we don't use `future`, [this page](https://python-future.org/compatible_idioms.html) contains a fairly comprehensive list of Python 2/3 compatibility sticking points to look out for.
```

Comment:
`When adding code the works with` - grammar

Reply:
nitpick: do you think we could present a really simple bullet point list of typical things which may not be obvious, that devs should do in order to ensure compatibility? with examples? You mention that _"it's advisable to use the `six` types for casting and comparisons"_, can we help busy coders by providing some concrete examples of what this typically means for us when writing code? I think that could be helpful and worthwhile.

Reply:
Grammar fixed, and added some examples of common Python 2/3 fixes.  Additionally, the link to the future documentation page contains some more, comprehensive examples; hopefully that's called out by the text around the link.